### PR TITLE
conditionally WebSerBundle, new sonata integration release

### PR DESCRIPTION
**Pull request by ElectricMaxxx**

### CODE REVIEW

**Comment by ElectricMaxxx** [comment]
Now it works fine and fluffy.

**Comment by ElectricMaxxx** [comment]
@dbu you know which environment is deployed to this plattform.sh ? I got a completely different behavior then local. All move routes got a `405 METHOD NOT ALLOWED` which is strange. I can not see if it comes from `CmfResourcesRestBundle` ' controller or from Symfony.

Btw: i think we/i need a next improvement on the tree-browser: movements within one collection (reorder only) should not do a move call – reorder only.

File: app/AppKernel.php [hunk 1]
```diff
@@ -70,6 +70,10 @@ public function registerBundles()
             new AppBundle\AppBundle(),
         ];
 
+        // Symfony 3.3 moved server:* commands to another bundle
```
Comment (dbu) [inline app/AppKernel.php]: the sandbox need not be generic. lets require that web server bundle and simply instantiate it

Comment (ElectricMaxxx) [inline app/AppKernel.php]: That won't work  i think. For Symfony < 3.3 the functions (`server:run`, ...) works on demand. From Symfony 3.3 those stuff comes from `WebServerBundle`, which does not exist befor. So what won't work, or i don't know how, is to require conditional dependencies. 
This way it works as Symfony requires it by its own.

Comment (dbu) [inline app/AppKernel.php]: imho we should simply update https://github.com/symfony-cmf/cmf-sandbox/blob/04b0e9876efce273af4db1b367c5cb7359a4fdb2/composer.json#L21 to say ^3.3. the sandbox is not about supporting as many versions as possible.

Comment (ElectricMaxxx) [inline app/AppKernel.php]: ahh ok, this travis does not iterate over many versions.

**Comment by dbu** [comment]
great! thanks a lot!

**Pull request merged**